src/CoreWCF.WebHttp/src/CoreWCF/Dispatcher/HelpOperationInvoker.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// TODO: Help
+
+using System.Threading.Tasks;
+using CoreWCF.Web;
+
+namespace CoreWCF.Dispatcher
+{
+    internal class HelpOperationInvoker : IOperationInvoker
+    {
```

Comment:
Is this still needed? We're not dispatching the help page through the CoreWCF Dispatcher

Reply:
I'm still using the WCF routing so the old help page URLs redirect to SwaggerUI.